packages/app/src/cli/utilities/extensions/configuration.ts

```diff
@@ -8,13 +8,14 @@ interface GetUIExensionResourceURLOptions {
 export function getUIExtensionResourceURL(
   uiExtensionType: UIExtensionTypes,
   options: GetUIExensionResourceURLOptions,
-): {url: string | undefined} {
-  switch (uiExtensionType) {
-    case 'checkout_ui_extension':
-      return {url: options.checkoutCartUrl}
-    case 'product_subscription':
-      return {url: options.subscriptionProductUrl ?? ''}
-    default:
-      return {url: ''}
+): {url: string} {
+  if (uiExtensionType === 'checkout_ui_extension' && options.checkoutCartUrl) {
+    return {url: options.checkoutCartUrl}
   }
+
+  if (uiExtensionType === 'product_subscription') {
+    return {url: options.subscriptionProductUrl ?? ''}
+  }
+
+  return {url: ''}
```

Comment:
I switched to if/else because the new types understand that `options.checkoutCartUrl` can be undefined, and that the url should never be undefined.